tools/github-page-script-injection/injectScript.js

```diff
@@ -31,7 +31,9 @@ let scriptContent = fs.readFileSync(scriptFilePath, 'utf8');
 // Replace the placeholder string with the actual connection string
 const connectionString = 'InstrumentationKey=1ae9e7ce-18f1-4e14-8fc0-acbf0ed28895;IngestionEndpoint=https://eastus-8.in.applicationinsights.azure.com/;LiveEndpoint=https://eastus.livediagnostics.monitor.azure.com/';
 let iKeyForCfgSync = "1ae9e7ce-18f1-4e14-8fc0-acbf0ed28895";
-scriptContent = scriptContent.replace(`connectionString: "YOUR_CONNECTION_STRING"`, `instrumentationKey: "${iKeyForCfgSync}",\n \t\textensionConfig: {\n\t\t\t"AppInsightsCfgSyncPlugin": {\n\t\t\t\tcfgUrl:"https://js.monitor.azure.com/beta/ai.config.1.cfg.json"\n\t\t\t}\n\t\t}`);
+//let cfgUrl = "https://js.monitor.azure.com/beta/ai.config.1.cfg.json";
+let cfgUrl = "https://tst2cdnstore.blob.core.windows.net/cdn/nevtest/ai.cfgSync.json";
```

Comment:
Lets not use this URL as it's pointing directly to the test storage account which won't do everything needed for every browser and won't really handle the scale correctly.

Let use the "test" folder of the standard `js.monitor.azure.com` (we will need to update the deployment pipeline for this -- or perhaps use the "nightly" folder and we just put in a "test" file (not called ai.cfgSync.json) perhaps ai.typedoc.cfgSunc.json) or our test CDN

Reply:
So current nightly is going to `https://[js.monitor.azure.com/nightly/ai.3-nightly3.min.js](https://js.monitor.azure.com/nightly/ai.3-nightly3.min.js)`

But I noticed that we are not currently publishing the cfgSync at all nightly, so we will need to add that to the pipeline and then also "add" the ability for it to publish a "test" version (so we don't accidently publish over the real one in the standard pipeline... (Which I was I suggested another file) so maybe either
- `ai_typedoc.cfgSunc.json` or
- `ai_test.cfgSync.json` (as the dot after the `ai` will cause issues parsing the name for promoting the version

>